src/WrapperApp/WrapperApp.tsx

```diff
@@ -52,15 +52,8 @@ function WrapperApp() {
 	}, [isAuthorized, tabsValue]);
 
 	useEffect(() => {
-		const tabTitles = {
-			login: 'Login',
-			editor: 'Editor',
-			inputFiles: 'Input files',
-			simulations: 'Simulations',
-			results: 'Results',
-			about: 'About'
-		};
-		document.title = `${tabTitles[tabsValue as keyof typeof tabTitles] || 'Yaptide'}`;
+		const result = tabsValue.replace(/([A-Z])/g, ' $1');
+		document.title = result.charAt(0).toUpperCase() + result.slice(1);
```

Comment:
Why `slice(1)` ?

Reply:
Because we want concatenate rest of the result string to the first letter, which is uppercased in the same line.

Reply:
The code is still far from being self-explanatory. Be more explicit in giving the comments, for example say that:

// below we try construct human readable name for document title which is displayed in web browser tab title
// to get it we transform the text containted in the tabsValue, which contain camelCase text with given tab name, for example inputFiles
// first we prepend a space before every capital letter, to split single camelCase word into several words separated by space, for example `inputFiles` become `input Files`
// then we make first character of first word to be upper case, so `input Files` becomes `Input Files`

Reply:
ChatGPT when asked about javascript code suggests two options:

1. use  a function:

```js
function camelCaseToNormalText(input) {
    // Insert a space before all caps and trim any leading space
    let result = input.replace(/([A-Z])/g, ' $1').trim();
    
    // Convert the first character to uppercase and the rest to lowercase
    return result.charAt(0).toUpperCase() + result.slice(1).toLowerCase();
}

// Example usage:
let camelCaseString = "helloWorldExample";
let normalText = camelCaseToNormalText(camelCaseString);
console.log(normalText); // Outputs: "Hello world example"
```

2. Use `lodash` library

I think that for such a small task first apprach is enough. Anyway the option with a function has several benefits:
 - name of the function conveys a message on the purpose of the code (convert camelCase to normal text)
 - comments in the function explain how the algorithm works

Reply:
> Maybe it will be worth to make this function in Utils segment and export it, to make it reusable in other parts of app? What do you think about it @grzanka?

Good idea !